test/integration/apiserver/field_validation_test.go

```diff
@@ -2893,3 +2978,541 @@ func setupCRD(t *testing.T, config *rest.Config, apiGroup string, schemaless boo
 
 	return crd
 }
+
+func BenchmarkFieldValidation(b *testing.B) {
+	defer featuregatetesting.SetFeatureGateDuringTest(b, utilfeature.DefaultFeatureGate, features.ServerSideFieldValidation, true)()
+	flag.Lookup("v").Value.Set("0")
+	server, err := kubeapiservertesting.StartTestServer(b, kubeapiservertesting.NewDefaultTestServerOptions(), nil, framework.SharedEtcd())
+	if err != nil {
+		b.Fatal(err)
+	}
+	config := server.ClientConfig
+	defer server.TearDownFn()
+
+	// don't log warnings, tests inspect them in the responses directly
+	config.WarningHandler = rest.NoWarnings{}
+
+	client := clientset.NewForConfigOrDie(config)
+
+	b.Run("Post", func(b *testing.B) { benchFieldValidationPost(b, client) })
+	//b.Run("Put", func(b *testing.B) { benchFieldValidationPut(b, client) })
```

Comment:
Why are these commented right now?

Reply:
I needed to fix them (see hold note above), but still wanted to post the working "Post" benchmark to show Alex how I was doing benchmarking.

I've fixed them now and uncommented them